doc/command_ref.rst

```diff
@@ -387,6 +387,11 @@ Options
 ``--showduplicates``
     Show duplicate packages in repositories. Applicable for the list and search commands.
 
+.. _transient_option-label:
+
+``--transient``
+    Applicable only on bootc systems. Perform transactions using a transient overlay which will reset on reboot. See also the :ref:`persistence <persistence-label>` configuration option.
```

Comment:
"Applicable only on bootc systems." looks strange to me.  Almost like bootc is a typo.  Given that this is documentation, I'd recommend something else along with 'bootc' to make it clear this is for systems managed by the bootc tool or something.

"which will reset on reboot" also comes across strangely.  I would say "which will be lost on the next reboot".

Reply:
Changed this to "Applicable only on bootc (bootable containers) systems." I think "bootc systems" is common enough in RHEL docs at this point: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/using_image_mode_for_rhel_to_build_deploy_and_manage_operating_systems/managing-kernel-arguments-in-bootc-systems#managing-kernel-arguments-in-bootc-systems.

IMO the best thing would be to link to https://github.com/containers/bootc but not sure whether that's appropriate for this man page.